mkdocs.yml

```diff
@@ -7,6 +7,8 @@ theme:
 
 nav:
   - Home: index.md
+  - 2022:
+    - General Info: 2022/index.md
```

Comment:
Just an FYI: you can skip updating `mkdocs.yml` to generate the new page but hide it from the ToC if you want to share a preview of a page internally.

Reply:
Ah, interesting, thanks. I don’t really care if anyone sees this before it’s 100% done … and no one will find it by accident, I think.